hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestHoodieClientMultiWriter.java

```diff
@@ -351,10 +352,10 @@ public void testHoodieClientMultiWriterWithClustering(HoodieTableType tableType)
         numRecords, 200, 2);
     client2.commit(newCommitTime, result2);
     // Schedule and run clustering while previous writer for commit 003 is running
-    SparkRDDWriteClient client3 = getHoodieWriteClient(cfg);
+    SparkRDDWriteClient client3 = getHoodieWriteClient(cfg3);
     // schedule clustering
     Option<String> clusterInstant = client3.scheduleTableService(Option.empty(), TableServiceType.CLUSTER);
-    assertFalse(clusterInstant.isPresent());
+    assertTrue(clusterInstant.isPresent());
```

Comment:
you might have conflicts when you rebase. just watch out

Reply:
rebased and resolved conflict.